Cargo.toml

```diff
@@ -1,39 +1,33 @@
-[package]
-name = "heroku-go-buildpack"
+[workspace]
+resolver = "2"
+members = [
+    "buildpacks/go",
+    "common/go-utils",
+]
+
+[workspace.package]
 version = "0.0.0"
-description = "Heroku Go Cloud Native Buildpack"
+rust-version = "1.75"
 edition = "2021"
 publish = false
-rust-version = "1.74"
 
-[lints.rust]
+[workspace.lints.rust]
 unreachable_pub = "warn"
 unsafe_code = "warn"
-# TODO: Enable this lint once the lib target is split out to a shared code crate,
-# which will reduce the false positives and make using this lint viable.
+# TODO: Enable this lint once the heroku-go-utils binary targets are split into
+# their own crates. This should reduce false positives and make this lint viable.
 # unused_crate_dependencies = "warn"
 
-[lints.clippy]
+[workspace.lints.clippy]
 panic_in_result_fn = "warn"
 pedantic = "warn"
 unwrap_used = "warn"
 
-[dependencies]
-flate2 = { version = "1", default-features = false, features = ["zlib"] }
-# libcnb has a much bigger impact on buildpack behaviour than any other dependencies,
-# so it's pinned to an exact version to isolate it from lockfile refreshes.
-libcnb = { version = "=0.17.0", features = ["trace"] }
-libherokubuildpack = { version = "=0.17.0", default-features = false, features = ["log"] }
-regex = "1"
-semver = "1"
+[workspace.dependencies]
```

Comment:
Dependabot doesn't currently handle updating `[workspace.dependencies]` well - so I would avoid using it for any external dependency that needs updating (it's fine to use it for any local `path = ` dependencies though). It also doesn't really offer a massive benefit in a Dependabot world, since it's not likely the versions will get out of sync between crates.

xref https://github.com/heroku/buildpacks-nodejs/pull/726 and https://github.com/heroku/buildpacks-jvm/pull/614

Reply:
Yeah, makes sense. Fixed in https://github.com/heroku/buildpacks-go/pull/200/commits/e3e5fec1178c3af5b663fc0eb7d650b07f323f92.